website/blog/2021-08-12-tungsten.md

```diff
@@ -0,0 +1,321 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.10.5 - Tungsten
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.10.5, which contains a lot of fixes, adds Scala 3.0.1 support,
+and even brings one nice feature!
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">150</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">71</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">18</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/40?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add support for `textDocument/selectionRange`
+- [Scala 3] `Inferred Type` code action and other improvements
+- Formatter - `runner.dialect` support and `.scalafmt.conf` update suggestions
+- Dependency sources - search symbol references in the workspace
+
+## Add support for `textDocument/selectionRange`
+
+Due to the great work by [@ckipp01](https://github.com/ckipp01)  now Metals implements `textDocument/selectionRange` LSP-method.
+It allows you to easily select an expression and expand it using a single shortcut combination.
+
+![gif1](https://user-images.githubusercontent.com/13974112/121028775-605be380-c7a8-11eb-99f1-948ce0b730ab.gif)
```

Comment:
I don't feel strongly about this, but it's a bummer that VS Code inserts an extra range in there that Metals doesn't give it. It's super obvious. What do you think of using a different example that doesn't add in the extra range?

![2021-07-12 20 10 54](https://user-images.githubusercontent.com/13974112/125335989-7a5f8780-e34d-11eb-911f-42f851478737.gif)

You can use this one if you want.

Reply:
Replaced by suggested gif.
Honestly, it's hard to spot this issue :smile:

Reply:
Haha I think I'm just paranoid about it and am afraid people will immediately notice it and be like "THIS DOESN'T WORK RIGHT!" 😆